### PR TITLE
Adds a bunch of dreams.

### PR DESCRIPTION
**Pull request by Belsima**
Mostly written by mineself, some ported from Eros, and a pair lifted from the discord. Also makes the current ones somewhat more consistent in grammar.

### CODE REVIEW

**Comment by Anewbe** [comment]
Do you compile and run these before you commit them?

**Comment by Belsima** [comment]
This one was made in the web thing because BYOND is still fucked. I was hoping I could at least touch dream files and have it work.

File: code/modules/flufftext/Dreaming.dm [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+	"the Head of Personnel","the Head of Security","the Chief Engineer","the Research Director","the Chief Medical Officer",
+	"the Detective","the warden","a member of the internal affairs","a station engineer","the Janitor","the Atmospheric Technician",
+	"the Quartermaster","a Cargo Technician","the Botanist","a Shaft Miner","the Psychologist","the Chemist","a Geneticist",
+	"the Virologist","the Roboticist","the Chef","the Bartender","the Chaplain","the Librarian","a mouse","an ERT member",
+	"a beach","the holodeck","a smoky room","a voice","the cold","a mouse","an operating table","the bar","the rain","a Skrell",
+	"an Unathi","a Tajaran","the Station Intelligence core","the mining station","the research station","a beaker of strange liquid",
+	"a Teshari", "a Diona nymph","the supermatter","Major Bill","a Morpheus ship with a ridiculous name","the Exodus","a star",
+	"a Dionaea gestalt","the chapel","a distant scream","endless chittering noises","glowing eyes in the shadows","an empty glass",
+	"a disoriented Promethean","towers of plastic","a Gygax","a synthetic","a Man-Machine Interface","maintenence drones",
```
Comment (mistyLuminescence) [inline code/modules/flufftext/Dreaming.dm]: "mainten**e**nce drones"

File: code/modules/flufftext/Dreaming.dm [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
-	"a beach","the holodeck","a smokey room","a voice","the cold","a mouse","an operating table","the bar","the rain","a skrell",
-	"an unathi","a tajaran","the ai core","the mining station","the research station","a beaker of strange liquid",
+	"the Head of Personnel","the Head of Security","the Chief Engineer","the Research Director","the Chief Medical Officer",
+	"the Detective","the warden","a member of the internal affairs","a station engineer","the Janitor","the Atmospheric Technician",
+	"the Quartermaster","a Cargo Technician","the Botanist","a Shaft Miner","the Psychologist","the Chemist","a Geneticist",
+	"the Virologist","the Roboticist","the Chef","the Bartender","the Chaplain","the Librarian","a mouse","an ERT member",
+	"a beach","the holodeck","a smoky room","a voice","the cold","a mouse","an operating table","the bar","the rain","a Skrell",
+	"an Unathi","a Tajaran","the Station Intelligence core","the mining station","the research station","a beaker of strange liquid",
+	"a Teshari", "a Diona nymph","the supermatter","Major Bill","a Morpheus ship with a ridiculous name","the Exodus","a star",
```
Comment (mistyLuminescence) [inline code/modules/flufftext/Dreaming.dm]: Shouldn't "the Exodus" be "the Northern Star" here?

Comment (Belsima) [reply code/modules/flufftext/Dreaming.dm]: It's along the lines of "the Luna" being higher up in the list.

File: code/modules/flufftext/Dreaming.dm [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
 	"riches","space","a crash","happiness","pride","a fall","water","flames","ice","melons","flying","the eggs","money",
-	"the head of personnel","the head of security","a chief engineer","a research director","a chief medical officer",
-	"the detective","the warden","a member of the internal affairs","a station engineer","the janitor","atmospheric technician",
-	"the quartermaster","a cargo technician","the botanist","a shaft miner","the psychologist","the chemist","the geneticist",
-	"the virologist","the roboticist","the chef","the bartender","the chaplain","the librarian","a mouse","an ert member",
-	"a beach","the holodeck","a smokey room","a voice","the cold","a mouse","an operating table","the bar","the rain","a skrell",
-	"an unathi","a tajaran","the ai core","the mining station","the research station","a beaker of strange liquid",
+	"the Head of Personnel","the Head of Security","the Chief Engineer","the Research Director","the Chief Medical Officer",
+	"the Detective","the warden","a member of the internal affairs","a station engineer","the Janitor","the Atmospheric Technician",
```
Comment (Anewbe) [inline code/modules/flufftext/Dreaming.dm]: Warden, Internal Affairs, and Station Engineer aren't capitalized.

**Pull request merged**